tests/unit/amo/components/TestBadge.js

```diff
@@ -1,45 +1,53 @@
-import { shallow } from 'enzyme';
 import * as React from 'react';
 
 import Badge from 'amo/components/Badge';
-import Icon from 'amo/components/Icon';
+import { render, screen } from 'tests/unit/helpers';
 
 describe(__filename, () => {
   it('renders a badge', () => {
-    const badge = shallow(<Badge label="super badge" />);
+    const label = 'badge label';
+    const { root } = render(<Badge label={label} />);
 
-    expect(badge).toHaveClassName('Badge');
-    expect(badge.find(Icon)).toHaveLength(0);
-    expect(badge.children()).toIncludeText('super badge');
+    expect(root).toHaveClass('Badge');
+    expect(screen.queryByClassName('Icon')).toHaveLength(0);
+    expect(screen.getByText(label)).toBeInTheDocument();
```

Comment:
would `ByLabelText` work here?

Reply:
Interesting idea, but `ByLabelText` seems to be intended only to work with actual `label`s: https://testing-library.com/docs/queries/bylabeltext/. It does not find anything in this component.